tests/unit/test-metadata-test.js

```diff
@@ -0,0 +1,31 @@
+import QUnit, { module, test } from 'qunit';
+import { getTestMetadata } from '@ember/test-helpers';
+import { TestMetadata } from '@ember/test-helpers/test-metadata';
+
+module('Test Metadata', function() {
+  test('getTestMetadata returns default test metadata', function(assert) {
+    let test = QUnit.config.current;
+    let testMetadata = getTestMetadata(test);
+
+    assert.ok(testMetadata instanceof TestMetadata);
+    assert.deepEqual(testMetadata.types, []);
+  });
+
+  module('Annotated Test Metadata', function(hooks) {
+    hooks.beforeEach(function() {
+      let test = QUnit.config.current;
```

Comment:
Same as above, we should probably use `this` instead of `QUnit.config.current` (since `current` is the `Test` instance, not the `this` context)

Reply:
Same as above.